pkg/apis/batch/types.go

```diff
@@ -532,6 +532,21 @@ const (
 	JobFailureTarget JobConditionType = "FailureTarget"
 )
 
+type JobReasonType string
+
+const (
```

Comment:
Maybe worth indicating which ones are beta in the comment and leaving a link to the kep (like we have in the kube_features.go file)

Reply:
good call.  Updated.